client/src/lib/sessions/hooks/useConfirmPracticeReminders.ts

```diff
@@ -0,0 +1,59 @@
+import {useCallback} from 'react';
+import {Alert} from 'react-native';
+import {useTranslation} from 'react-i18next';
+import dayjs from 'dayjs';
+import utc from 'dayjs/plugin/utc';
+import {useNavigation} from '@react-navigation/native';
+import {NativeStackNavigationProp} from '@react-navigation/native-stack';
+
+import usePracticeRemindersSetting from '../../reminders/hooks/usePracticeRemindersSetting';
+import {ModalStackProps} from '../../navigation/constants/routes';
+import {
+  calculateNextHalfHour,
+  thisWeekday,
+} from '../../reminders/utils/timeHelpers';
+
+dayjs.extend(utc);
```

Comment:
I believe it's better to put this in `lib/i18n/index.ts` as we have all other extend there. Also are you 100% sure we need this?

Reply:
I will move it to index and toggle a bit to see if it's really needed but I think so.